route/index.js

```diff
@@ -57,4 +58,6 @@ router.put('/:id', async (req, res, next) => {
   }
 });
 
-module.exports = router;
\ No newline at end of file
+indexRouter.use('/puppies', puppyRouter)
```

Comment:
Your api was serving up puppies at the URI `/api/`. For example, `GET` `/api/` would serve up an array of all puppies.

While not strictly wrong, this is pretty unconventional, and we would expect to get the puppies at a URI like `/api/puppies/`.

I implemented that by taking your router (with GET / POST / PUT / DELETE routes for `/` and `/:id`) and "mounted" that router onto the `/puppies` path. Now you have to request `GET` `/api/puppies` to see all the pups.